build-scripts/create-archive.sh

```diff
@@ -6,6 +6,9 @@ source $(dirname "$(readlink -f "$0")")/create-spec.sh
 # Prepare source archive
 [[ -d rpmbuild/SOURCES ]] || mkdir -p rpmbuild/SOURCES
 
+# Mark current directory as safe for git
+git config --global --add safe.directory $(pwd)
+
```

Comment:
I'd prefer to have this command at a single place for all invocations, so I suggest to remove it from here and from https://github.com/eclipse-bluechi/bluechi/blob/main/build-scripts/version.sh#L23 and place it in https://github.com/eclipse-bluechi/bluechi/blob/main/build-scripts/create-spec.sh#L4 before version fetching

Reply:
Unfortunately, this will introduce the same issue (not safe dir) when `version.sh` is invoked from meson directly to fetch the version. What do you think about moving it in `version.sh` so its always executed?